.travis.yml

```diff
@@ -6,31 +6,36 @@ env:
     else git log -1 --skip 1 --pretty=format:%ct; fi)
 sudo: false
 language: ruby
-cache: bundler
+cache:
+  bundler: true
+  yarn: true
 branches:
   only:
-  - master
-  - /^v\d+\.\d+\.\d+$/
+    - master
+    - /^v\d+\.\d+\.\d+$/
 services:
-- postgresql
-- elasticsearch
+  - postgresql
+  - elasticsearch
 addons:
   chrome: stable
 before_install:
-- gem update --system
+  - gem update --system
+install:
+  - bundle install --path vendor/bundle
```

Comment:
Did you try this without the explicit install steps? I __think__ should perform these steps when it sees a `Gemfile.lock` and `yarn.lock`

Reply:
I did try without and the build failed. I think we both read the same doc, but the auto-yarn only works if the language is node.